src/lib/RoomConnection.ts

```diff
@@ -211,7 +212,13 @@ export function handleStreamAdded(
     }
     // screenshare
     return new RoomEvent("screenshare_started", {
-        detail: { participantId: clientId, stream, id: streamId, isLocal: false },
+        detail: {
+            participantId: clientId,
+            stream,
+            id: streamId,
+            isLocal: false,
+            hasAudioTrack: stream.getAudioTracks().length > 0,
```

Comment:
unsure if this is correct... input welcome

Reply:
I think this should be sufficient. It at least answers the question the prop seemingly asks `hasAudioTrack`. However, the track could be disabled, meaning no audio, but I honestly dont know if that is the case event for a screenshare started without audio. Probably your test should be sufficient 👍

Reply:
I guess semantically `hasAudioTrack` would still be true in this case even if it was disabled or muted right?

But yeah, we can find out if it becomes an issue I guess

Reply:
Yep, we are staying true to the prop name 👍